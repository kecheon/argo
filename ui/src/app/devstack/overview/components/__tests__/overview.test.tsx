import '@testing-library/jest-dom/extend-expect';
// import {render} from '@testing-library/react';
import {shallow} from 'enzyme';
import * as React from 'react';
import Overview from '../overview';

describe('Overview component', () => {
    it('renders the component', () => {
        const component = shallow(<Overview />);
        expect(component).toMatchSnapshot();
    });
});