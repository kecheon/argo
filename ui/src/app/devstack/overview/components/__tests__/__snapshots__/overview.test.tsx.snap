// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overview component renders the component 1`] = `ShallowWrapper {}`;
